server/src/common/score-base.entity.ts

```diff
@@ -0,0 +1,79 @@
+import { Column } from 'typeorm';
+import { CustomBaseEntity } from '../common/custom-base.entity';
+
+export class ScoreBaseEntity extends CustomBaseEntity {
+  @Column({ type: 'int', nullable: true })
```

Comment:
Final score and comments can only be columns, other columns can be inside the score/data as a JSONB!

Reply:
Should've captured this earlier. It's my fault!